.gitignore

```diff
@@ -57,4 +57,10 @@ docs/_build/
 .vscode
 
 #virtual environment
-venv/
\ No newline at end of file
+venv/
+
+# node modules
+node_modules
+
+# javascript linter
+.eslintrc.yml
```

Comment:
you should actually check in the lintrc. Everyone should be linting against the same standard.

Reply:
fixed